src/report.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "Weekly report"
+date: "`r format(Sys.Date(), format = '%B %d, %Y')`"
+output: 
+  html_fragment:
+  html_document:
+    keep_md: yes
+    self_contained: false
+include-in-header:
+    file: includes/in-header.html
+---
+
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
+```
+
+```{r preliminaries}
+library(hubData)
+library(ggplot2)
+library(plotly)
+library(crosstalk)
+theme_set(theme_bw())
+
+this_ref_date <- as.Date("2024-12-04")
+quantiles <- c(0.05, 0.1, 0.25, 0.5, 0.75, 0.9, 0.95)
+```
+
```

Comment:
Since this is a fragment, the date header will not appear. This will display the time this was generated (set to UTC)

```suggestion

**This report was generated on `r format(Sys.time(), tz = "UTC", format = '%B %d, %Y %H:%M:%S')`.**

```